C++/straw.cpp

```diff
@@ -494,8 +494,8 @@ map<int32_t, indexEntry> readMatrixHttp(CURL *curl, int64_t myFilePosition, cons
     membuf sbuf(buffer, buffer + size);
     istream bufin(&sbuf);
 
-    int32_t c1 = readInt32FromFile(bufin);
-    int32_t c2 = readInt32FromFile(bufin);
+    readInt32FromFile(bufin);
```

Comment:
Let's not remove the variable name for future readability.

Reply:
That was to fix a compiler warning (CRAN doesn't allow warnings or significant notes without explanation). How about we keep it as an inline comment, e.g. `/* int32_t c1 = /* readInt32FromFile(bufin);`?

Reply:
@sa501428 Does the latest commit satisfy your concern?